src/core/manager.c

```diff
@@ -1899,7 +1899,7 @@ static void manager_preset_all(Manager *m) {
         CLEANUP_ARRAY(changes, n_changes, install_changes_free);
 
         log_info("Applying preset policy.");
-        r = unit_file_preset_all(RUNTIME_SCOPE_SYSTEM, /* file_flags = */ 0,
+        r = unit_file_preset_all(RUNTIME_SCOPE_SYSTEM, /* flags = */ 0,
```

Comment:
Rather than changing here. Please rename `flags` -> `file_flags` in shared/install.h .
As you can see, shared/install.c uses `file_flags`.